packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -255,25 +309,54 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
           isSelected,
           onClick: () => {
             !option.disabled && onChange?.(option.value);
-            close();
+            !multiple && close();
+          },
+          ref: (el: HTMLElement) => {
+            if (isSelected && !multiple) {
+              el?.scrollIntoView();
```

Comment:
No way to make it work? Maybe save first selected index or something

Reply:
Amm... I tried, but it is getting triggered on every new item select...
So I would need to add some more logic and make it scroll only on open or something :?

Reply:
Yeah... then you would need to save whether it is first render, maybe enough for this time 😀

Reply:
For some reason this seems to work on multiple although it is disabled.

Reply:
Yes. In single select, it also works, if I remove that line 😄